src/distribution/chi.rs

```diff
@@ -446,6 +446,12 @@ mod test {
     }
 
     #[test]
+    #[should_panic]
+    fn test_mode_panic() {
```

Comment:
The convention I've been following is to name these panic cases for why they're panicking, e.g. `test_mode_freedom_lt_one`. If we could follow that, that would be great

Reply:
Good to know this, thanks!

Reply:
Done!